src/sqlfluff/core/dialects/dialect_ansi.py

```diff
@@ -142,6 +142,17 @@
     ]
 )
 
+# Set the value table functions. These are functions that, if they appear as
+# an item in "FROM', are treated as returning a COLUMN, not a TABLE. Among
+# Apparently, among dialects supported by SQLFluff, only BigQuery has this
+# concept, but this set is defined in the ANSI dialect because:
+# - It impacts core linter rules (see L020 and several other rules that subclass
+#   from it) and how they interpret the contents of table_expressions
+# - At least one other database (DB2) has the same value table function,
+#   UNNEST(), as BigQuery. DB2 is not currently supported by BigQuery.
```

Comment:
`not currently supported by sqlfluff`?

Reply:
Ha, good catch. Fingers faster than 🧠. 😄